aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/AbstractAnalyseMojo.java

```diff
@@ -69,6 +69,14 @@ public abstract class AbstractAnalyseMojo extends AbstractAemMojo {
     @Parameter(defaultValue = "true", property = "failon.analyser.errors")
     private boolean failOnAnalyserErrors;
 
+    /**
+     * If enabled, warnings for maven plugin version or SDK API version are turned
+     * into errors and fail the build.
+     * @since 1.6.2
+     */
+    @Parameter(defaultValue = "false", property = "aem.analyser.strict.version")
+    protected boolean strictVersionValidation;
```

Comment:
I think it makes sense to be able to configure the sensitivity of each analyzer (and the version check) individually.

Reply:
Don't disagree - but I think this approach is ok for the version checks done by the plugin itself - these are not analyser checks and therefore are not executed in Cloud Service.
We can have a follow up issue about specific analysers. I think we also have a issue in Apache Sling open for this already.